Gems/EMotionFX/Code/EMotionFX/Rendering/OpenGL2/Source/GLRenderUtil.cpp

```diff
@@ -139,10 +133,10 @@ namespace RenderGL
     // destroy the allocated memory
     void GLRenderUtil::CleanUp()
     {
-        for (uint32 i = 0; i < MAX_LINE_VERTEXBUFFERS; ++i)
+        for (VertexBuffer*& lineVertexBuffer : mLineVertexBuffers)
```

Comment:
Is this working with mLineVertexBuffers being a pointer?

Reply:
`mLineVertexBuffers` is declared as:
```cpp
VertexBuffer*               mLineVertexBuffers[2]{};
```
That is a C-style array of `VertexBuffer*`s. C++11 range-for loops work on c-style arrays.

We could of course rewrite this to use an `AZStd::array<VertexBuffer*, 2>`, the usage of that member would be the same.

Reply:
It is fine, hopefully the code will be gone in a bit anyway.